### PR TITLE
Fixed report bugs

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by andrewconner** [comment]
@2is10 / @eishay please review

File: shoebox/app/com/keepit/common/analytics/MongoEventStore.scala [hunk 1]
```diff
@@ -38,11 +38,11 @@ case class MongoSelector(eventFamily: EventFamily) {
     this
   }
   def withMetaData(field: String) = {
-    q = q ++ ("metaData.%s".format(field) $exists true)
+    q = q ++ ("metaData.metaData.%s".format(field) $exists true)
```
Comment (eishay) [inline shoebox/app/com/keepit/common/analytics/MongoEventStore.scala:41]: didn't you had a plan to rename `metaData.metaData` to avoid confusion ?

Comment (andrewconner) [inline shoebox/app/com/keepit/common/analytics/MongoEventStore.scala:41]: Yeah, I mentioned that. It'll involve migrating all the old data and changing the extension, so I didn't do it now.

**Comment by 2is10** [comment]
looks okay to me

**Pull request merged**